pkg/apiserver/profiling/router.go

```diff
@@ -100,10 +101,19 @@ func (s *Service) getGroupList(c *gin.Context) {
 	c.JSON(http.StatusOK, resp)
 }
 
+type ResponseTask struct {
+	ID                uint                    `json:"id" gorm:"primary_key"`
+	State             TaskState               `json:"state" gorm:"index"`
+	Target            model.RequestTargetNode `json:"target" gorm:"embedded;embedded_prefix:target_"`
+	Error             string                  `json:"error" gorm:"type:text"`
+	StartedAt         int64                   `json:"started_at"` // The start running time, reset when retry. Used to estimate approximate profiling progress.
+	ProfileOutputType string                  `json:"profile_output_type"`
```

Comment:
As you already have this field in the db model, then this struct can be omitted (that you can use the db model directly) as there is no new information need to be added. We can have this when we need something not in the db but want to be passed to the frontend.